### PR TITLE
Issue #4858 use values instead of module constants in update hooks

### LINKED ISSUE #4858: Errors in update hooks from use of undefined constants
**Description of the bug**

Getting undefined constant errors in update hooks. For example:

> Warning: Use of undefined constant COMMENT_NODE_OPEN - assumed 'COMMENT_NODE_OPEN' (this will throw an Error in a future version of PHP) in comment_update_1001() (line 357 of /app/core/modules/comment/comment.install).

> Warning: Use of undefined constant COMMENT_NODE_HIDDEN - assumed 'COMMENT_NODE_HIDDEN' (this will throw an Error in a future version of PHP) in comment_update_1003() (line 401 of /app/core/modules/comment/comment.install).

> Warning: Use of undefined constant VIEWS_STORAGE_OVERRIDE - assumed 'VIEWS_STORAGE_OVERRIDE' (this will throw an Error in a future version of PHP) in node_update_1018() (line 1194 of /path/to/backdrop/core/modules/node/node.install).

**Steps To Reproduce**

To reproduce the behavior:

1. Upgrade a Drupal 7 site to Backdrop (can test with just core modules).
2. Run update.php

**Additional information**

Add any other information that could help, such as:

- Backdrop CMS version: 1.17.4
- PHP 7.2

[Updated to include more examples. Also see https://github.com/backdrop/backdrop-issues/issues/4343]

### PR DESCRIPTION
**Pull request by herbdool**
Fixes https://github.com/backdrop/backdrop-issues/issues/4858

### CODE REVIEW

File: core/modules/comment/comment.install [hunk 1]
```diff
@@ -354,13 +355,13 @@ function comment_update_1001() {
   foreach ($node_types as $node_type) {
     // Move the comment variables into each node type's config.
     $settings = array(
-      'comment_default' => update_variable_get('comment_' . $node_type->type, COMMENT_NODE_OPEN),
+      'comment_default' => update_variable_get('comment_' . $node_type->type, 2), // COMMENT_NODE_OPEN
```
Comment (indigoxela) [inline core/modules/comment/comment.install:358]: Just a minor coding standards suggestion, I'm not sure if you're using phpcs and what your attitude regarding these standards is (sorry for being picky).

phpcs nags: "Comments may not appear after statements."

Hint: The inline comments should be on their own line, start with a capital letter, end with a full-stop, exclamation mark, or question mark (and shouldn't be longer than 80 characters).

The comments explaining the "magic numbers" are per se a good idea!

Comment (herbdool) [reply core/modules/comment/comment.install:358]: I found a lot of other similar comments in the code so figured it was ok. I looked at Backdrop coding standards and recall reading that for lists it's ok to put the comment at the end.

Comment (herbdool) [reply core/modules/comment/comment.install:358]: At least that's how I read:

> If each line of a list needs a separate comment, the comments may be given on the same line and may be formatted to a uniform indent for readability.

Comment (indigoxela) [reply core/modules/comment/comment.install:358]: OK, then we should really update the phpcs coder sniffer variant to validate the stuff that's actually documented. :wink:
Feel free to mark this discussion as resolved (I can't).

**Comment by ghost** [comment]
Thanks guys! Merged into 1.x and 1.18.x.

**Pull request merged**